pkg/bootstrap/config.go

```diff
@@ -67,7 +67,7 @@ const (
 	// "reporter" prefix is for istio standard metrics.
 	// "component" suffix is for istio_build metric.
 	v2Prefixes = "reporter=,"
-	v2Suffix   = ",component"
+	v2Suffix   = ",component,istio"
```

Comment:
Does this impact users that have the flag false? I mean I know it changes the bootstrap, but I am guessing `istio` doesn't match any metrics so its a NOP, right?

Reply:
No, this only affects new "istio" metrics.